platform-tests/src/test/java/org/junit/platform/launcher/core/LauncherFactoryTests.java

```diff
@@ -22,15 +22,36 @@
 import org.junit.platform.commons.util.ReflectionUtils;
 import org.junit.platform.launcher.Launcher;
 import org.junit.platform.launcher.LauncherDiscoveryRequest;
+import org.junit.platform.launcher.TestExecutionListener;
 import org.junit.platform.launcher.TestIdentifier;
 import org.junit.platform.launcher.TestPlan;
+import org.junit.platform.launcher.listener.CustomTestExecutionListener;
 
 /**
  * @since 1.0
  */
 class LauncherFactoryTests {
 
 	@Test
+	void testCustomTestExecutionListenerServiceIsLoaded() {
```

Comment:
Please don't use the "test" prefix for `@Test` methods, even if the class already has such naming in place. 😉